doc/source/whatsnew/v0.21.0.txt

```diff
@@ -155,7 +155,7 @@ Indexing
 
 - When called with a null slice (e.g. ``df.iloc[:]``), the ``.iloc`` and ``.loc`` indexers return a shallow copy of the original object. Previously they returned the original object. (:issue:`13873`).
 - When called on an unsorted ``MultiIndex``, the ``loc`` indexer now will raise ``UnsortedIndexError`` only if proper slicing is used on non-sorted levels (:issue:`16734`).
-
+- Fixed ``TimedeltaIndex.get_loc`` ``for np.timedelta64`` inputs (:issue:`16909`).
```

Comment:
Close, but not quite.  Change "``for np.timedelta64``" to "handling of ``np.timedelta64``"